docs/_ext/conversion.py

```diff
@@ -73,57 +73,76 @@ def create_r_returns(doc_list):
 
     return doc_str
 
+pattern_start = re.compile(r'>>> |^\.\.\. ')
+# pattern_methods = re.compile(r'([A-Za-z_]\d*)\.([A-Za-z_]\d*)')
+pattern_methods = re.compile(r'([A-Za-z_]\d*)\.([A-Za-z_]\d*)(?=(?:[^"]|"[^"]*")*$)(?=(?:[^\']|\'[^\']*\')*$)')
```

Comment:
This attempts to not replace `.`s that are inside quotes.

Reply:
I do not understand what you mean? The regular expression logic essentially decides the parity of how many quotes are to the right of the dot. So it could be replaced by ad-hoc code.

Reply:
Sorry, you're right, thought about it wrong. Then this looks good!